lib/barcelona/plugins/pcidss_plugin.rb

```diff
@@ -238,6 +413,180 @@ def build_resources
             tag("barcelona-role", "pcidss")
           ]
         end
+
+        add_resource("AWS::EC2::Volume", "OSSECManagerVolume") do |j|
+          j.AvailabilityZone "ap-northeast-1a"
```

Comment:
`district.stack_resources["SubnetTrusted1"]` can't be used here because `AvailabilityZone` of `AWS::EC2::Volume` is zone identifier but `VPCZoneIdentifier` of `AWS::AutoScaling::AutoScalingGroup` is subnet ID. AZ of ASG's VPC zone identifier (subnet ID) and AZ of EC2 volume must match